apps/payment/views.py

```diff
@@ -52,7 +50,23 @@ def payment(request):
 
                     payment_object.handle_payment(request.user)
 
-                    _send_payment_confirmation_mail(payment_relation)
+                    receipt = PaymentReceipt(
+                        to_mail=payment_relation.user.email,
+                        from_mail=payment_relation.payment.responsible_mail(),
+                        subject="[kvittering] " + payment_relation.payment.description(),
+                        description=payment_relation.payment.description(),
+                        transaction_date=payment_relation.datetime
+                    )
+                    receipt.save()
+
+                    item = ReceiptItem(
+                        receipt=receipt,
+                        name=payment_relation.payment.description(),
+                        price=int(payment_relation.payment_price.price)
+                    )
+                    item.save()
+
+                    _send_receipt(receipt)
```

Comment:
If adding support for automagically sending PaymentReceipts as mentioned in models, this might be redundant ;-)